app/src/db/models/submission.ts

```diff
@@ -0,0 +1,88 @@
+import { Prisma } from '@prisma/client';
+import { disconnectRelation, user } from '.';
+import { fromYrn, toYrn } from '../../components/utils';
+
+import type { IStamps } from '../../interfaces/IStamps';
+import type { ChefsSubmissionForm } from '../../types';
+
+// Define types
+const _submission = Prisma.validator<Prisma.submissionDefaultArgs>()({});
+const _submissionWithRelations = Prisma.validator<Prisma.submissionDefaultArgs>()({
+  include: { user: true }
+});
+
+type UserRelation = {
+  user:
+    | {
+        connect: {
+          userId: string;
+        };
+      }
+    | {
+        disconnect: boolean;
+      };
+};
+type DBSubmission = Omit<Prisma.submissionGetPayload<typeof _submission>, 'assignedToUserId' | keyof IStamps> &
+  UserRelation;
+
+type Submission = Prisma.submissionGetPayload<typeof _submissionWithRelations>;
```

Comment:
If we ever need to start referring to these types outside of just this model DTO we'll need to move this into the types folder.